Tasks/Gradle/testlib/testlib-main.ts

```diff
@@ -0,0 +1,416 @@
+// Replaces vsts-task-lib for testing purposes.
```

Comment:
Implements vsts-task-lib for test environments, allowing task inputs and variables to be set up ahead of time without the need for the JSON-based structure seen in the TaskRunner system. Filesystem calls are functional for use in test. Keeps track of errors, warnings and debug logging messages.